coordinator/src/main/scala/filodb.coordinator/DatasetCoordinatorActor.scala

```diff
@@ -132,6 +133,7 @@ private[filodb] class DatasetCoordinatorActor(projection: RichProjection,
   var flushesStarted = 0
   var flushesSucceeded = 0
   var flushesFailed = 0
+  var rowsIngested = 0L
```

Comment:
Hmmm.   So have a mutable case class for tracking stats, and an immutable version?

`case class MutableStats(var flushesStarted: Int, var flushesSucceeded: Int)`